core/src/test/java/com/linecorp/armeria/internal/common/PathAndQueryTest.java

```diff
@@ -162,6 +165,84 @@ void doubleDotsInNameValueQuery() {
         });
     }
 
+    private static String normalizedQuery(String string) {
+        return string.replaceAll("%2[eE]", ".")
+                     .replaceAll("%2f", "%2F");
+    }
+
+    @Test
+    void allowDoubleDotsInNameValueQuery() throws UnsupportedEncodingException {
```

Comment:
nit; where is `UnsupportedEncodingException` thrown?

```suggestion
    void allowDoubleDotsInNameValueQuery() {
```

Reply:
It is cruft. Should be removed.